runtime/common/src/parachains.rs

```diff
@@ -91,6 +90,18 @@ impl<N: Saturating + One + PartialOrd + PartialEq + Clone> Iterator for BlockNum
 pub trait ParachainCurrency<AccountId> {
 	fn free_balance(para_id: ParaId) -> Balance;
 	fn deduct(para_id: ParaId, amount: Balance) -> DispatchResult;
+	fn transfer_in(
```

Comment:
This trait needs some documentation. I don't know who the source and destination are supposed to be in practice. Is one of them a parachain account?